### ISSUE TITLE
Problème en combat

### ISSUE THREAD

**Issue by Skoueek** [Skinz3/Symbioz#16]
Salut, j'ai une erreur lorsque je suis en combat, je tape et ça me fait une erreur.
http://image.noelshack.com/fichiers/2016/46/1479242922-huhu.png
Je peux vous apporter plus de détails si vous voulez. :)

**Comment by nojo123** [comment]
Hi sir, my French is quite bad so i hope it won't be a problem for you that i comment in English. I think that after the new update the damage handler broke as I'm having a similar error when in a fight. Did the error occur after using the god mode? If so maybe that is the problem, hopefully Matspyder51 can have a look at it.

**Comment by Matspyder51** [comment]
J'ai vu l'erreur hier, c'est parce que je test si le joueur est en god mais il teste aussi quand on tape un monstre donc il faut que je modifie la condition

**Comment by Skoueek** [comment]
nojo123 You don't need to upload the old version of the emulator, you can soon download it on github.
Matspyder51 Ok, merci de ta réponse très rapide, bonne chance.

**Comment by nojo123** [comment]
Alright :P, hope its getting a fix soon.

**Comment by Skoueek** [comment]
Dit nous quand le changement sera fait :)

**Comment by irissian** [comment]
how to actiuve god mode?

**Comment by nojo123** [comment]
Just type .god. Its buggy right now tho

**Comment by AndroKami** [comment]
Salut , Voilà j'ai le même problème ! comment puis-je avoir l'ancienne version ?

**Comment by nojo123** [comment]
https://mega.nz/#!dAlRGLga!0vuAhEBUvOsynquXbCDSQQNrZ0JGSLWhtIkc0kOFmvU
This one has no problems, I added the addcaracs (changed to .addskill) command manually. Also the PA and PM limit has been removed i've been trying to get it back but i can't find it anywhere, so if someone finds it please comment :)

**Comment by AndroKami** [comment]
Thanks u so much !!!

**Comment by nojo123** [comment]
If you want i can upload every major update i make like adding commands or fixing certain bugs, just let me know if you're interested :)

**Comment by AndroKami** [comment]
Yeah yeah sure i m really interested !! Thank u so much again !!!!

**Comment by albidroid** [comment]
Thank you! :)
